src/bundles/Elsa.Server.Web/Enums/ApplicationRole.cs

```diff
@@ -0,0 +1,8 @@
+namespace Elsa.Server.Web;
+
+public enum ApplicationRole
+{
+    Hybrid,
+    Api,
+    Worker
```

Comment:
Is there no configuration needed for Workers? E.g. disabling the Http module?

Reply:
I considered that, but since the load balancer needs to be configured to not route HTTP requests to this role anyway, there's no need. This keeps it simple, but we can do it if necessary for some reason.